LICENSE.md

```diff
@@ -1,6 +1,6 @@
 The MIT License (MIT)
 
-Copyright (c) 2016
+Copyright (c) 2018
```

Comment:
I believe that copyright years are not supposed to be updated, and if they do it should be a year range.

Reply:
Oh yeah, good catch. An artifact of `ember-cli-update`.